### PR TITLE
Update PVR API 6.2.0

### PR DESCRIPTION
**Pull request by phunkyfish**
v2.4.2 Update PVR API 6.2.0

Do not merge yet, part of API change

### CODE REVIEW

**Comment by palinek** [comment]
@phunkyfish I needed to release a fix before a while.... please, increase the version number in this PR accordingly. Thank you.

**Comment by phunkyfish** [comment]
No problem 👍

File: src/Data.cpp [hunk 1]
```diff
@@ -226,6 +226,9 @@ void Data::ReleaseUnneededEPG()
         // notify about the epg change...and delete it
         EPG_TAG tag;
         memset(&tag, 0, sizeof(EPG_TAG));
+        tag.iSeriesNumber = -1;
```
Comment (palinek) [inline src/Data.cpp]: Shouldn't we use directly your's PR `EPG_TAG_INVALID_SERIES_EPISODE` constant here?

**Comment by phunkyfish** [comment]
Yes. We will. One step at a time 😃

**Comment by phunkyfish** [comment]
API changes merged in kodi, please merge this and create a release. Otherwise, the breaking change can effect your users who run kodi nightlies.

ios-armv7 error unrelated.

**Comment by palinek** [comment]
Done... thank you.

**Pull request merged**